library/Fission/Web/Auth.hs

```diff
@@ -1,30 +1,30 @@
 module Fission.Web.Auth
-  ( server
-  , context
+  ( Checks
+  , server
   , basic
   , user
   , checkUser
   ) where
 
 import Crypto.BCrypt
-import Database.Selda
+import Database.Esqueleto
```

Comment:
Protip: you generally only want to import `Database.Esqueleto`, since it re-exports a big chunk of `Database.Persist`.